.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Description
+
+Please provide a brief description of the changes in this pull request.
+
+### Motivation
+
+Explain the motivation behind these changes and why they are necessary.
+
+### Type of Change
+
+- [ ] Bug fix (non-breaking change that resolves an issue)
+- [ ] Feature enhancement (non-breaking change that adds functionality)
+- [ ] Documentation update (non-breaking change that improves documentation)
+- [ ] Refactoring (non-breaking change that improves code structure)
+- [ ] Other (please specify)
+
+### Checklist
+
+Please review the following checklist and ensure that all items are completed before submitting the pull request:
+
+- [ ] My changes do not introduce any breaking changes or change existing API behaviors.
+    - Keep in mind, There are rather large users of lol-html so even small changes to API behavior can be breaking
+      for them, and will be considered breaking for lol-html. Lol-html has a commitment to stability; breaking
+      changes are only to be considered as a last resort.
+- [ ] I have tested my changes thoroughly and they do not cause any regressions.
+- [ ] My code follows the project's coding style guidelines.
```

Comment:
Would be nice to provide a link to those guidelines (though, I don't remember if we have them lol)